### DISCUSSION TITLE
How can we launch url every time when pdf is opened

### DISCUSSION THREAD

**Discussion by palak-sethia** [py-pdf/pypdf#715] [Q&A]
Hi @mstamy2 , 
Can we edit existing pdf and make change in that such as we navigate to a URL each time when pdf is opened.

-Thanks 
Palak

**Comment by mstamy2** [comment]
Perhaps try using the [addJS](https://pythonhosted.org/PyPDF2/PdfFileWriter.html#PyPDF2.PdfFileWriter.addJS) method.
